aws-terraform/ecs/README.md

```diff
@@ -0,0 +1,35 @@
+# Deploying the 1Password SCIM Bridge in AWS ECS with Terraform
+
+This document describes deploying the 1Password SCIM bridge to your Amazon Web Service Elastic Container Service Fargate using Terraform. It's just a suggested starting point - you may be using different services for different things, this example uses only AWS products. Please familiarize yourself with [Preparation.md](../../Preparation.md) before beginning.
+
+Prerequisites
+- [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) and [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- DNS Zone in Route 53
+- scimsession file and bearer token ([follow step 1 here](https://support.1password.com/scim/))
+- an empty file `terraform.tfvars` in this directory
```

Comment:
I made this the way I would prefer follow it, adding lines in each step. If you think this way makes more sense for more people, I can change it

Reply:
What I was thinking is instead of saying 'create an empty file' either, provide the file already empty or with "template"
or use locals instead of variables 

```terraform
locals  {
     # The domain you wish your SCIM bridge to be contacted over
     domain_name = FILL_THE_BLANKS
     dns_zone_id = FILL_THE_BLANKS
     aws_logs_group = FILL_THE_BLANKS
}
```

Reply:
```suggestion
```

Let's already provide this file.